notify/impl.go

```diff
@@ -987,45 +1013,39 @@ func (n *OpsGenie) Notify(ctx context.Context, as ...*types.Alert) (bool, error)
 
 		apiURL = n.conf.APIURL + "v2/alerts"
 		var teams []map[string]string
-		for _, t := range strings.Split(string(tmpl(n.conf.Teams)), ",") {
+		for _, t := range safeSplit(string(tmpl(n.conf.Teams)), ",") {
```

Comment:
I think this should be returned to just `strings.Split` and then check each `t` for being an empty string. If someone enters `"team1,"` in their config, it'll still end up adding `"name":""` in the json.

Reply:
done